### PR TITLE
fix bug for insert vertex | edge

### PR DESCRIPTION
**Pull request by bright-starry-sky**
fixed bugs as below:
Repeated vid or edge , for example delete vertex "a", "a"

### CODE REVIEW

File: src/executor/mutate/DeleteExecutor.cpp [hunk 1]
```diff
@@ -58,6 +58,9 @@ folly::Future<Status> DeleteVerticesExecutor::deleteVertices() {
     if (vertices.empty()) {
         return Status::OK();
     }
+    std::sort(vertices.begin(), vertices.end());
+    vertices.erase(unique(vertices.begin(), vertices.end()), vertices.end());
```
Comment (yixinglu) [inline src/executor/mutate/DeleteExecutor.cpp:62]: Need not to sort the vids, can use set to dedup them.

Comment (bright-starry-sky) [reply src/executor/mutate/DeleteExecutor.cpp:62]: > Need not to sort the vids, can use set to dedup them.

if using std::set , still need to convert it from std::set to std::vector. and it's not a good idea to change the parameter type of storage clients.

Comment (Shylock-Hg) [reply src/executor/mutate/DeleteExecutor.cpp:62]: I think you could add a `dedup` executor directly.

Comment (yixinglu) [reply src/executor/mutate/DeleteExecutor.cpp:62]: Agree to add dedup plan node in execution plan.

**Pull request closed without merging**